apps/system/js/app_window.js

```diff
@@ -223,14 +223,18 @@
     this.getTopMostWindow()._setActive(enable);
   };
 
-  AppWindow.prototype.inScope = function(scope) {
+  AppWindow.prototype.getScope = function() {
+    var url = new URL(this.config.url);
+    var scope = this.webManifest && this.webManifest.scope;
+    return scope || url.origin;
+  };
+
+  AppWindow.prototype.inScope = function(testScope) {
```

Comment:
Not sure why this change? We want to test that the current url is in the scope that we are checking (the one sent by parameter). i.e
inScope ('http://test.com') with url = 'http://test.com/test/bla' -> true
inScope ('http://test.com/test2') with url =  'http://test.com/test/bla' -> false